CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## [v0.20.0](https://github.com/mongodb/go-client-mongodb-atlas/tree/v0.20.0) (December 16, 2022)
+
+[Full Changelog](https://github.com/mongodb/go-client-mongodb-atlas/compare/v0.19.0...v0.20.0)
+
+**Merged pull requests:**
+
+- Added missing fields 'Links', 'CopySettings' and 'DeleteCopiedBackups… [\#332](https://github.com/mongodb/go-client-mongodb-atlas/pull/332) ([sangramPeerI](https://github.com/sangramPeerI))
```

Comment:
```suggestion
- Added missing fields 'Links', 'CopySettings' and 'DeleteCopiedBackups' [\#332](https://github.com/mongodb/go-client-mongodb-atlas/pull/332) ([sangramPeerI](https://github.com/sangramPeerI))
```

Think the ... is a typo

Reply:
autogenerated, it's picked from the commit message

Reply:
Thanks TIL